### PR TITLE
Storage class fixes and updates

### PR DESCRIPTION
**Pull request by miminar**

### CODE REVIEW

File: install_config/persistent_storage/topics/glusterfs_storageclass.adoc [hunk 1]
```diff
@@ -6,30 +6,37 @@ apiVersion: storage.k8s.io/v1
...
 provisioner: kubernetes.io/glusterfs
-parameters:
+parameters: <1>
   resturl: "http://127.0.0.1:8081" <1>
   restuser: "admin" <2>
   secretName: "heketi-secret" <3>
   secretNamespace: "default" <4>
   gidMin: "40000" <5>
   gidMax: "50000" <6>
+  volumeoptions: features.cache-invalidation on, nl-cache on <7>
```
Comment (miminar) [inline install_config/persistent_storage/topics/glusterfs_storageclass.adoc]: `volumetype` seems to be another good candidate. Any other suggestions?

**Comment by kalexand-rh** [comment]
@jhou1, will you PTAL?

File: install_config/persistent_storage/topics/glusterfs_storageclass.adoc [hunk 2]
```diff
@@ -6,30 +6,37 @@ apiVersion: storage.k8s.io/v1
 metadata:
   name: slow
 provisioner: kubernetes.io/glusterfs
-parameters:
+parameters: <1>
   resturl: "http://127.0.0.1:8081" <1>
```
Comment (jhou1) [inline install_config/persistent_storage/topics/glusterfs_storageclass.adoc]: This is the second `<1>` here, you possibly wanted `<2>`.

**Comment by jhou1** [comment]
LGTM given my comment is addressed.

**Comment by miminar** [comment]
Thanks. Addressed comments, added ` volumetype` parameter and updated link.

**Comment by kalexand-rh** [comment]
Thank you @jhou1!

@miminar, please squash your commits so I can merge. :)

**Comment by miminar** [comment]
@kalexand-rh squashed as you wished. Thank you.

**Comment by kalexand-rh** [comment]
Thank you, @miminar! Merging.

**Comment by vikram-redhat** [comment]
@kalexand-rh - the tests had passed, I think the UI is just not updating it.

**Comment by kalexand-rh** [comment]
@vikram-redhat, when I looked, GitHub wasn't running any builds. I'm glad it's finally caught up! Merging.

**Pull request merged**